docs/source/user_guide/automl.ipynb

```diff
@@ -342,37 +342,6 @@
     "best_pipeline.predict(X)"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Saving AutoMLSearch and pipelines from AutoMLSearch\n",
-    "\n",
-    "There are two ways to save results from AutoMLSearch. \n",
-    "\n",
-    "- You can save the AutoMLSearch object itself, calling `.save(<filepath>)` to do so. This will allow you to save the AutoMLSearch state and reload all pipelines from this.\n",
-    "\n",
-    "- If you want to save a pipeline from AutoMLSearch for future use, you can pickle the resulting pipeline. While pipeline classes themselves have a `.save(<filepath>)` method, you can also pickle the resulting pipeline in order to save it.\n",
-    "\n",
-    "** Stacked Ensembling pipelines cannot currently be pickled"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# saving the best pipeline using .save()\n",
-    "# best_pipeline.save(\"file_path_here\")\n",
```

Comment:
`.save` should still work right? Maybe we should add back this code snippet, but I agree we should delete `pickle.dumps`.

Reply:
Agreed, `save` still works because it uses `cloudpickle`, not `pickle`